test/scale.time.tests.js

```diff
@@ -395,29 +378,94 @@ describe('Time scale tests', function() {
 						type: 'time',
 						position: 'bottom'
 					}],
-					yAxes: [{
-						id: 'yScale0',
-						type: 'linear',
-						position: 'left'
-					}]
 				}
 			}
 		});
 
 		var xScale = chart.scales.xScale0;
 
-		expect(xScale.getPixelForValue('', 0, 0)).toBeCloseToPixel(71);
-		expect(xScale.getPixelForValue('', 6, 0)).toBeCloseToPixel(452);
-		expect(xScale.getPixelForValue('2015-01-01T20:00:00')).toBeCloseToPixel(71);
+		it('should be bounded by midnights', function() {
```

Comment:
[nit] Is the goal of this test to ensure that that when days are displayed, the last tick is at the end of the last day and similarly for the first tick?

After reading the several years ones, I think a better title might be "should be bounded by the nearest day beginnings" since I was a little unclear on what this meant at first.